website/src/app/page.tsx

```diff
@@ -19,14 +19,24 @@ export default function Page() {
     <>
       <section className="bg-neutral-100 pt-24">
         <div className="px-4 py-8 mx-auto max-w-screen-xl text-center lg:py-16 lg:px-12">
-          <h1 className="mb-4 md:text-6xl text-5xl justify-center font-extrabold tracking-tight leading-none text-neutral-900">
-            Blazing-fast alternative to legacy VPNs
+          <h1 className="mb-8 md:text-6xl text-5xl justify-center inline-block font-extrabold tracking-tight leading-none text-neutral-900">
+            Secure remote access.
+            <span className="ml-2 text-primary-450 underline inline-block">
+              That scales.
+            </span>
           </h1>
-          <h2 className="mb-8 text-xl tracking-tight text-neutral-800 sm:px-16 xl:px-48">
-            Manage secure remote access to your company’s most valuable services
-            and resources with Firezone. We’re open source, and built on
-            WireGuard®, a modern protocol that’s up to 4-6x faster than
-            OpenVPN.
+          <h2 className="mb-8 text-xl tracking-tight justify-center font-medium text-neutral-900 sm:px-16 xl:px-48 inline-block">
+            Firezone is a fast, flexible VPN replacement built on WireGuard®
+            that integrates with your identity provider and requires{" "}
```

Comment:
Perhaps front-load the zero-conf by saying:

> Firezone is a fast, flexible and zero-conf VPN built on WireGuard.

Reply:
It is a bit of a stretch because there is _some_ configuration you need to do (set up sites etc). But maybe this can be the goal :blush:

Reply:
> Firezone is a fast, flexible and zero-conf VPN built on WireGuard.

I've always found that mentioning 3 things has a nice ring to it!

Reply:
My 2c :)

Reply:
I think what Thomas proposed is good.

If you wanted to leave it at 2 adjectives, maybe say
```
Firezone is a fast and flexible VPN replacement built on Wireguard
```

Also, should we throw in something about zero trust?  Something like:
```
Helping you move from VPN to Zero Trust every step of the way
```
Just a thought that popped in my head.  No worries if it doesn't fit what you're going for.

Reply:
> there is some configuration

True, but not in the traditional sense I think (config files). It's a little close to some of Tailscale's marketing but I think I can work it in.

> should we throw in something about zero trust

Yeah I was considering that. I think most people still don't know what it means, so I was thinking "VPN replacement" gets the point across